libr/asm/arch/mips/gnu/mips-dis.c

```diff
@@ -620,6 +620,16 @@ parse_mips_dis_option (const char *option, unsigned int len)
   val = option + (optionlen + 1);
   vallen = len - (optionlen + 1);
 
+  if (strncmp ("abi", option, optionlen) == 0
```

Comment:
swap conditions ( == is faster than strncmp, and you can save some strncmps when the len doesnt matches. Also, use ! instead of ==0